config/Ruby/pry_repl.rb

```diff
@@ -1,6 +1,7 @@
 require 'rubygems'
 gem 'pry'
```

Comment:
Why not just use `gem 'pry', '=0.9.12.6'`?

Reply:
As of Oct 25, 2017, the file pry_repl.rb in the current distribution of SublimeREPL (as installed by Package Control today) has not been edited since Mar 14, 2013.

In particular, these changes have not been made. If you are reading this because you Googled "Pry::InputCompleter" after being unable to get SublimeREPL to work with Ruby, then make these changes manually to your copy.

Reply:
September 7, 2018, the above is still true. Apply these changes to `config/Ruby/pry_repl.rb` and `config/Rails/pry_repl.rb` locally and this package will work. Tested with `Rails 5.2.1` and `Ruby 2.5.1`.